### DISCUSSION TITLE
v1.3.0-preview1

### DISCUSSION THREAD

**Discussion by gabbybilka** [microsoft/WindowsAppSDK#3488] [Announcements]
## 1.3 Preview 1 (1.3.230228005-preview1) 🔃

The first preview of WinAppSDK 1.3 is now available! Check out the [release notes](https://aka.ms/windowsappsdk/1.3/1.3-preview1/release-notes) for more information on the new & updated features and known issues.

### Updating to this version
The 1.3-preview1 package is available at https://www.nuget.org/packages/Microsoft.WindowsAppSDK/1.3.230228005-preview1

### Providing feedback
As we introduce new & upcoming features in our preview releases, we appreciate all of your feedback as we work towards a stable release:
- For WinUI 3 feedback, please open an issue on the [WinUI repo](https://github.com/microsoft/microsoft-ui-xaml/issues/new/choose).
- For feedback regarding all other Windows App SDK components, please open an issue here, on the [Windows App SDK repo](https://github.com/microsoft/WindowsAppSDK/issues/new/choose).

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WindowsAppSDK/releases/tag/v1.3.0-preview1'>v1.3.0-preview1</a>.</em>

**Comment by shelllet** [comment]
great！

**Comment by bogdan-patraucean** [comment]
Any other changes than what was part of the experimental release?
